romancal/source_catalog/tests/test_psf.py

```diff
@@ -147,3 +139,106 @@ def test_azimuthally_smooth():
 
     assert np.mean(delta) < 1.0e-6
     assert mad_std(delta) < 1.0e-8
+
+
+# new routines: render_stamp, _get_jitter_params, _evaluate_gaussian_fft, add_jitter
+def test_render_stamp(setup_inputs):
+    # some basic tests that we can render a psf
+    _, psf_model = setup_inputs
+    stamp = psf.render_stamp(2000, 2000, psf_model, 19)
+    assert 0.9 < np.sum(stamp) < 1.1
+    assert stamp.shape[0] == stamp.shape[1] == 19
+    stamp = psf.render_stamp(0, 0, psf_model, 19)
+    assert 0.9 < np.sum(stamp) < 1.1
+    stamp = psf.render_stamp(0, 0, psf_model, 1)
+    assert np.sum(stamp) < 0.6
+    assert stamp.shape[0] == stamp.shape[1] == 1
+
+
+def test_get_jitter_params():
+    from types import SimpleNamespace
+
+    meta = SimpleNamespace()
+    res = psf._get_jitter_params(meta)
+    assert res["jitter_major"] > 0
+    assert res["jitter_minor"] > 0
+    assert np.isfinite(res["jitter_position_angle"])
+    meta.jitter_major = 4
+    res = psf._get_jitter_params(meta)
+    assert res["jitter_major"] == meta.jitter_major
```

Comment:
```suggestion
    res = psf._get_jitter_params({})
    assert res["jitter_major"] > 0
    assert res["jitter_minor"] > 0
    assert np.isfinite(res["jitter_position_angle"])
    meta = {"jitter_major": 4}
    res = psf._get_jitter_params(meta)
    assert res["jitter_major"] == meta["jitter_major"]
```
To update the test for the previous suggestion.